docs/structure.ts

```diff
@@ -383,6 +383,17 @@ export const STRUCTURE = [
           },
```

Comment:
@dimaatkaev please re-generate the documentation so that the output.json is also included.